.gitignore

```diff
@@ -36,7 +36,7 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
-.coverage
+.coverage.*
```

Comment:
There are still `.coverage` files in the main directory which should be removed.

Reply:
Yes; `.gitignore` was updated after these coverage files showed up. I'll have to delete them from the remote branch directly.